web/static/css/app.css

```diff
@@ -512,3 +512,61 @@ code {
 lm-ls1 {
   letter-spacing: 0.1rem;
 }
+
+@media screen and (min-width: 1210px) {
+  .homepage_video_container {
+    width: 500px;
+    height: 280px;
+  }
+}
+
+@media screen and (min-width: 1121px) and (max-width: 1209px) {
+  .homepage_video_container {
+    width: 450px;
+    height: 252px;
+  }
+}
+
+@media screen and (min-width: 1040px) and (max-width: 1122px) {
+  .homepage_video_container {
+    width: 400px;
+    height: 224px;
+  }
+}
+
+@media screen and (min-width: 60em) and (max-width: 1039px) {
+  .homepage_video_container {
+    width: 350px;
+    height: 196px;
+  }
+}
+
+@media screen and (max-width: 60em) {
```

Comment:
I used pixels to get the exact screen widths given to me by chrome when resizing the screen and the black lines on video appear

`em`s were used to match the exact break points for tachyon media query implementations